packages/utilities/src/warn/warnMutuallyExclusive.ts

```diff
@@ -10,7 +10,7 @@ import { ISettingsMap, warn } from './warn';
 export function warnMutuallyExclusive<P>(componentName: string, props: P, exclusiveMap: ISettingsMap<P>): void {
   if (typeof process !== 'undefined' && process.env.NODE_ENV !== 'production') {
     for (const propName in exclusiveMap) {
-      if (props && propName in props) {
+      if (props && propName in props &&  props[propName] !== undefined) {
```

Comment:
The code could be simplified further like this:
```suggestion
      if (props && props[propName] !== undefined) {
```